src/main/components/form/validator.ts

```diff
@@ -300,3 +300,17 @@ export const isValidCurrency: Validator = value => {
     return 'minLengthRequired';
   }
 };
+
+let otherGenderTitle: string | string[];
+
+export const validateGenderTitle: Validator = value => {
+  otherGenderTitle = value;
+};
+
+export const validatePreferredOther: Validator = value => {
+  if (otherGenderTitle === 'Other') {
+    if (/^\d+$/.test(value as string) || /[0-9]/.test(value as string) || (value as string).trim().length === 0) {
```

Comment:
Please also review sonarcloud for code coverage and smells - https://sonarcloud.io/summary/new_code?id=et-sya&pullRequest=255

Reply:
Added validator test